content/design.fr.md

```diff
@@ -0,0 +1,172 @@
+{
+    "title": "Guide de conception et mise en page",
+    "type": "design-guide",
+    "date": "2019-09-29T14:42:25+02:00",
+    "last_edited": "2019-09-29T14:42:25+02:00",
+    "authors": [ "baltpeter" ],
+    "tags": [ "datenanfragen.de", "datarequests.org", "demandetesdonnees.fr", "guide de conception", "guide de mise en page" ]
+}
+
+Afin de rendre nos sites Web, documents, etc. cohérents et accessibles, le projet demandetesdonnees.fr a défini (et continuera à définir) un ensemble de directives de conception et de mise en page.  
+Le but de cette page est de documenter et d'expliquer ces lignes directrices.
+
+<div class="box box-warning">
+Ce guide n'en est qu'à ses débuts. Ni notre charte de conception ni la documentation n'ont été finalisées. Attendez-vous à ce qu'elles changent toutes les deux à l'avenir.<br>
+</div>
+
+## Couleurs
+
+Nous avons défini une palette de couleurs principale et une palette étendue. Combinées au noir pur et au blanc pur, ces deux palettes constituent la sélection concluante des couleurs utilisées pour demandetesdonnees.fr. Aucune autre couleur ne peut être utilisée (pas même pour les illustrations).
+
+For convenience, the color palette is available as an [Adobe Swatch Exchange (ASE) file](/downloads/datenanfragen.ase) and an [Adobe Creative Cloud library](https://shared-assets.adobe.com/link/4a911d28-c227-4821-475f-4c7efe2039d1).
```

Comment:
This can be deleted.

Reply:
Oh sorry, I wasn't taking about the whole page. Just this one left-over English sentence. :D 

I reverted the change.